mern/client/src/__tests__/DAO_Tests/deleteSession.test.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import deleteSessionDAO from "../../utils/DAO/deleteSessionDAO";
+
+jest.mock("axios");
```

Comment:
Can we add documentation somewhere on how to run the JEST tests? Either in the code or in the README?